packages/react/src/components/Tag/Tag.tsx

```diff
@@ -15,40 +15,23 @@ export type TagProps = {
   /** Color of the tag
    * @default neutral
    */
-  color?: BrandColor | VariantColor;
+  color?: VariantColor | BrandColor;
```

Comment:
Suggest just removing this union and list them all together in one string literal. Its not really necessary anymore, it will also show the correct order in the select in Storybook.